microsite/data/plugins/dev-quotes.yaml

```diff
@@ -0,0 +1,10 @@
+---
+title: Dev Quotes
+author: Peter Macdonald
+authorUrl: https://github.com/Parsifal-M
+category: Humor
+description: Displays a coding/progamming related quote designed as a footer for the Homepage, although to be honest it can be used anywhere you like!
+documentation: https://github.com/Parsifal-M/backstage-dev-quotes?tab=readme-ov-file#dev-quotes-homepage
```

Comment:
It would be great to include an image in your documentation, it really helps people who are looking at using your plugin get a feel for its functionality

Reply:
Done! :smile: